### PR TITLE
Feature/tidy up template

### PR DESCRIPTION
**Pull request by cezarpopa**
Hi,
i've prepared the following PR which adds the following set of changes:

1. Make all template strings translatable
2. Use short array notation
3. Fix heading elements not being in a sequentially-descending order
4. Check whether a post has a featured image set before trying to display it
5. Add the mandatory alt attribute to image tags
6. Standardise spacing/alignment across all twig views
7. Introduce a editorconfig config file.

Thank you,
Cezar

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# http://make.wordpress.org/core/handbook/coding-standards/
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = tab
```
Comment (szepeviktor) [inline .editorconfig:14]:
```suggestion
indent_style = tab
indent_size = 4
```

GitHub displays tabs based on this info 👁️

**Comment by szepeviktor** [comment]
Very nice pull request.

File: .editorconfig (hunk 1 above)
Comment (cezarpopa) [reply .editorconfig:14]: I can amend/remove that if it needed.

Comment (szepeviktor) [reply .editorconfig:14]: It would be nice to set `indent_size`. Here TAB characters are 8 wide!

**Pull request closed without merging**